rubicon/objc/objc.py

```diff
@@ -1314,6 +1326,65 @@ def __setattr__(self, name, value):
                 super(ObjCInstance, type(self)).__setattr__(self, name, value)
 
 
+class ObjCListInstance(ObjCInstance):
```

Comment:
~~It might be useful to inherit from `collections.abc.Sequence` here. It provides default implementations for most standard Python sequence methods, like `find`, `count`, etc.~~

PS: Nevermind, that would cause name conflicts. `-[NSArray count]` returns the number of objects in the array, but Python `list.count` is a method that counts how often an object is present in a list.